app/Console/Commands/SendTestPullRequestWebhookCommand.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Http\Api\Controllers\HandleGitHubPullRequestWebhookController;
+use App\Http\Kernel;
+use Illuminate\Console\Command;
+use Illuminate\Http\Request;
+
+class SendTestPullRequestWebhookCommand extends Command
```

Comment:
I might want to remove this one? It's very useful for local testing though…

Reply:
Maybe put it in a namespace `Illuminate\Console\Command\Testing`?